packages/styled-system/src/css-var.ts

```diff
@@ -0,0 +1,186 @@
+import { Dict, isCssVar, isObject, pick } from "@chakra-ui/utils"
+import analyzeBreakpoints from "./breakpoints"
+import type { CSSMap, WithCSSVar } from "./types"
+
+const separator = ">"
+
+const replace = (value: string, str = ".") => value.replace(/>/g, str)
+
+export const toVar = (value: string) => `var(${value})`
```

Comment:
What do you think about adding a prefix for all chakra CSS variables to prevent name clashes with existing ones?